src/Randomizer.App/Windows/GenerateRomWindow.xaml

```diff
@@ -198,6 +198,12 @@
             </StackPanel>
           </Expander>
 
+          <Expander Header="Metroid Controls"
```

Comment:
Maybe we should reconsider how we display the options, since the list keeps growing. I'll have to think of something.

Reply:
Yeah. The generator window has been getting pretty unwieldy, both from a code perspective and user perspective. The code part is why I opted to split this out into a new user control. Will probably do the same when I add the starting inventory UI stuff.

But yeah, the complexity of the generator window from a user perspective is something I've been thinking of as well. I know OoT Rando uses different tabs for the different settings. VARIA basically has you go through two pages: one for the logic, then another for the customization. I feel like neither are a great solution, but not sure I have any ideas for anything better really.